python/pyspark/sql/streaming.py

```diff
@@ -315,9 +315,9 @@ def load(self, path=None, format=None, schema=None, **options):
         :param schema: optional :class:`pyspark.sql.types.StructType` for the input schema.
         :param options: all other string options
 
-        >>> json_sdf = spark.readStream.format("json")\
-                                       .schema(sdf_schema)\
-                                       .load(tempfile.mkdtemp())
+        >>> json_sdf = spark.readStream.format("json") \\
```

Comment:
Yes, it seems it does. I took a look and a sweep before - https://github.com/apache/spark/pull/14063

I will take another look across the documentation again tomorrow.